builder/builder.go

```diff
@@ -1317,24 +1317,7 @@ gpgcheck=0
 priority=1
 {{end}}`
 
-// BuildBundles will attempt to construct the bundles required by generating a
-// DNF configuration file, then resolving all files for each bundle using dnf
-// resolve and no-op installs. One full chroot is created from this step with
-// the file contents of all bundles.
-func (b *Builder) BuildBundles(template *x509.Certificate, privkey *rsa.PrivateKey, signflag bool) error {
-	// Fetch upstream bundle files if needed
-	if err := b.getUpstreamBundles(b.UpstreamVer, true); err != nil {
-		return err
-	}
-
-	// Generate the dnf config file if it does not exist.
-	// This takes the template and adds the relevant local rpm repo path if needed
-	fmt.Println("Building bundles...")
-
-	timer := &stopWatch{w: os.Stdout}
-	defer timer.WriteSummary(os.Stdout)
-
-	timer.Start("BUILD BUNDLES")
+func (b *Builder) newConfIfNeeded() error {
```

Comment:
I think it would be better to call this `newDnfConfIfNeeded()`. We have too many things called `conf` in mixer, it helps to avoid confusion if things a a bit better defined.

Reply:
agree.